lib/ansible/modules/network/f5/bigip_pool.py

```diff
@@ -222,370 +177,564 @@
       state: "absent"
```

Comment:
@gundalow this is a common arg

https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/f5_utils.py#L187